### ISSUE TITLE
[Bug] Project fails to build due to incorrect STREQUAL usage in a submodule's CMake project

### ISSUE THREAD

**Issue by davidebeatrici** [firebase/firebase-cpp-sdk#652]
I'm using CMake 3.18.4.

```
CMake Error at firebase-cpp-sdk/external/src/firestore-build/external/src/abseil-cpp/CMake/AbseilHelpers.cmake:178 (message):
  Invalid build type: static
Call Stack (most recent call first):
  firebase-cpp-sdk/external/src/firestore-build/external/src/abseil-cpp/absl/base/CMakeLists.txt:42 (absl_cc_library)
```

The issue was fixed in abseil/abseil-cpp@22771d471930ce88e1e75d0ca9dd8c65a7b0f895.

**Comment by wu-hui** [comment]
Thanks..we will upgrade absl version to fix this.

**Comment by jonsimantov** [comment]
> Thanks..we will upgrade absl version to fix this.

@wu-hui Did this upgrade happen in the iOS SDK?

**Comment by b0bben** [comment]
Is this fixed in 8.8.0? Can't seem to find any info.
